Documentation/ApiOverview/SoftReferences/Index.rst

```diff
@@ -8,14 +8,32 @@ Soft References
 ===============
 
 "Soft References" are references to database elements, files, email
-addresses, URLs etc. which are found inside text fields. The
-:html:`<a href="t3://page?[page_id]>` tag for page links found in bodytext fields is an example of this.
+addresses, URLs etc. which are found inside text fields.
+
+For example, `tt_content.bodytext` can contain soft references to pages,
+to content elements, to files etc. The page reference can look
+like this:
+
+.. code-block:: html
+
+   <a href="t3://page?uid=1">link to page 1</a>
+
+In contrast to this, the field `pages.shortcut` contains the page id
+of a shortcut. This is a reference, but not a *soft* reference.
 
 The Soft Reference parsers are used by the system to find these
 references and process them accordingly in import/export actions and
 copy operations. Also, the soft references are used by integrity
-checking functions.
+checking functions. For example, when you try to delete a page,
+TYPO3 will warn you if there are incoming page links to this page.
+
+The soft references - as well as the ordinary references - are
+written to the soft reference index (table `sys_refindex`).
```

Comment:
Here "soft" should be removed, the index is for all references after all.

Reply:
Rephrased it